pages/food-groups/index.tsx

```diff
@@ -0,0 +1,280 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { useRef } from 'react'
+import FoodGroups from 'components/FoodGroups'
+
+import Auth from '@components/Auth'
+import { Button, Modal } from '@components/Custom'
+import { ACHIEVEMENT, useFirebase } from '@components/FirebaseContext'
```

Comment:
There are some major changes made to `FirebaseContext`. Please update the codes that are using the values and functions from this context.